src/platform-includes/debug-symbols-apple/_default.mdx

```diff
@@ -123,3 +124,67 @@ export SENTRY_URL=https://mysentry.invalid/
 ```
 
 </Alert>
+
+## Uploading source context
```

Comment:
Could we split this into its own page? This page is already long and enabling source context isn't required to get debug symbols working.

Reply:
Yes, I believe we can.